itscalledsoccer/client.py

```diff
@@ -43,7 +43,7 @@ def __init__(
             ]:
                 self.LOGGER.setLevel(getLevelName(logging_level.upper()))
             else:
-                print(f"Logging level {logging_level} not recognized!")
+                self.LOGGER.info(f"Logging level {logging_level} not recognized!")
```

Comment:
Have you come across [`loguru`](https://github.com/Delgan/loguru)? I've started using it in all my projects — cleans up a lot of this boilerplate.

Reply:
I have not! I'll make a feature request to switch to it